pkg/api/v1alpha1/cluster.go

```diff
@@ -32,6 +32,8 @@ const (
 	ClusterKind              = "Cluster"
 	RegistryMirrorCAKey      = "EKSA_REGISTRY_MIRROR_CA"
 	podSubnetNodeMaskMaxDiff = 16
+	rollingUpdate            = "RollingUpdate"
+	inPlaceUpgrade           = "InPlace"
```

Comment:
These should be in cluster_types.go somehow as enum right? That way we get the validation from that type checking?